apis/Owners.ts

```
@@ -360,6 +360,33 @@ export class Owners<
+   * @response `403` `void`
+   * @response `404` `void`
+   * @response `405` `void`
+   * @response `500` `void`
+   */
+  createFoodTruckDocumentPresignedUrls = (
+    data: ImageRequest,
+    params: RequestParams = {},
+  ) =>
+    this.request<CreateFoodTruckDocumentPresignedUrlsData, void>({
+      path: `/owners/me/food-truck-documents/images`,
+      method: "POST",
```

Biome (2.1.2)

[error] Duplicate class member name "createFoodTruckDocumentPresignedUrls"



(lint/suspicious/noDuplicateClassMembers)